src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/data/AppointmentMappingDto.kt

```diff
@@ -0,0 +1,33 @@
+package uk.gov.justice.digital.hmpps.nomisvisitsmappingservice.data
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import io.swagger.v3.oas.annotations.media.Schema
+import jakarta.validation.constraints.NotBlank
+import jakarta.validation.constraints.Size
+import uk.gov.justice.digital.hmpps.nomisvisitsmappingservice.jpa.AppointmentMapping
+import java.time.LocalDateTime
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Schema(description = "NOMIS to Appointment mapping")
+data class AppointmentMappingDto(
+
+  @Schema(description = "Appointment instance id", required = true)
```

Comment:
very very minor, can't this just be appointmentId throughout ? I think for sentencing we settled on prefixing the nomis key with "nomis" and just not prefixing the other key

Reply:
Well they have various appointment tables - see https://dsdmoj.atlassian.net/wiki/spaces/SAA/pages/4247093255/Appointments+database+tables
I thought Id match the name of their column